### ISSUE TITLE
Page Title

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#146]
Can we remove the versioning information from the page title in prototypes? Perhaps put this information in a comment? 
Ideally a page title should be the Sim's name.

**Comment by jessegreenberg** [comment]
I will ask other developers on Thursday why version id is part of the title, and if it can be removed.  The sim title is automatically generated for internationalization.

At the very least we can remove 'accessible-instance' from the title.  Published sims have the version number in the title like 'Balloons and Static Electricity 1.1.2'.  @terracoda would this be acceptable or would this also be confusing for screen reader users?

**Comment by terracoda** [comment]
@jessegreenberg that should work. The shorter the better. The page title is the first thing a screen reader user hears. A short, unique, relevant and meaningful page title is the goal. It is most important for when we have active interviews.

**Comment by jessegreenberg** [comment]
For the BASE prototypes, the title is set to "Balloons and Static Electricity 1.2.0" with 'accessible-instance' removed.  Closing this issue, if we need to adjust the title further, we can create a new issue to discuss removing version ID as well.

**Comment by jessegreenberg** [comment]
Reopening.  The built version of the sim has the old title.

**Comment by jessegreenberg** [comment]
For prototypes, dev info in the version id can be removed from build.  Closing.
